cms/envs/common.py

```diff
@@ -1020,6 +1020,9 @@
 ##### EMBARGO #####
 EMBARGO_SITE_REDIRECT_URL = None
 
+##### custom vendor plugin variables #####
```

Comment:
Perhaps say also that "JavaScript code can access this data using `JSON.parse(process.env.JS_ENV_EXTRA_CONFIG)`", and mention that one use case is enabling custom TinyMCE plugins.

Reply:
This has been addressed in [`e882cd4 - Adds more information about the JS_ENV_EXTRA_CONFIG env setting`](https://github.com/edx/edx-platform/pull/25695/commits/e882cd41a7c5a25f94bfe15d42fc478ed37469cd) 👍🏼